CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Update Redmine to version 5.1.3
+  - common_mark markdown formatter is set for new installations, markdown formatter is deprecated.
```

Comment:
In light of a missing formatter migration, I would suggest to phrase it somewhat like this:

```suggestion
  - Redmine 5.1.2 deprecates the previous markdown formatter. The formatter `common_mark markdown` will be set for new installations as well as for upgrading instances that use the old markdown formatter
```

Reply:
The migration was changed to warnings and a lot of notices because the Github-flavored markdown formatter does not fully drop-in-replaces the deprecated markdown formatter. Underlined text is no longer supported. In this light, the migration is kind of dangerous because lots of text may format differently which may not be any customer's delight.

Reply:
For new instances this should be the default though.